gapic-generator-cloud/templates/cloud/wrapper_gem/readme.erb

```diff
@@ -0,0 +1,95 @@
+# Ruby Client for the <%= gem.title %> API
+
+<%= gem.description %>
+
+<%= gem.summary %>
+
+Actual client classes for the various versions of this API are defined in
+_versioned_ client gems, with names of the form `<%= gem.name %>-v*`.
+The gem `<%= gem.name %>` is a convenience wrapper library that brings the
+verisoned gems in as dependencies, and provides high-level methods for
+constructing clients.
+
+View the [Client Library Documentation](<%= gem.library_documentation_url %>)
+for this library, <%= gem.name %>, to see the convenience methods for
+constructing client objects. Reference documentation for the client objects
+themselves can be found in the client library documentation for the versioned
+client gems:
+<%- last_index = gem.versioned_gems.size - 1 -%>
+<%- gem.versioned_gems.each_with_index do |name, i| -%>
+<%- sep = i == last_index ? "." : "," -%>
+[<%= name %>](https://googleapis.dev/ruby/<%= name %>/latest)<%= sep %>
+<%- end -%>
+
+<%- if gem.product_documentation_url -%>
+See also the [Product Documentation](<%= gem.product_documentation_url %>)
+for more usage information.
+
+<%- end -%>
+## Quick Start
+
+```
+$ gem install <%= gem.name %>
+```
+
+In order to use this library, you first need to go through the following steps:
+
+1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+<%- if gem.free_tier? -%>
+1. This API has a free tier. You may not need to
+   [enable billing](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+   to get started.
+<%- else -%>
+1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+<%- end -%>
+<%- if gem.api_id -%>
+1. [Enable the API.](https://console.cloud.google.com/apis/library/<%= gem.api_id %>.googleapis.com)
+<%- end -%>
+1. {file:AUTHENTICATION.md Set up authentication.}
+
+<%- if gem.migration? %>
```

Comment:
Optional for the greenfield libraries? if pre_migration_version is nil perhaps?

Reply:
This is exactly what `gem.migration?` tests: whether the library is being migrated as opposed to greenfield. (And yes, its implementation is essentially, is the migration version nil or not.)